doc/tutorial/tutorial.mdk

```diff
@@ -143,16 +161,29 @@ the [source code on github] using these [instructions].
 [source code on github]: http://github.com/FStarLang/FStar
 [instructions]: https://github.com/FStarLang/FStar/blob/master/INSTALL.md
 
-You can edit F\* code using your favorite text editor, but [Emacs],
-[Atom], and [Vim] have extensions that add special support for F\*,
-including syntax highlighting and interactive development. More
-details on [editor support] on the [F\* wiki].
+You can edit F\* code using your favorite text editor, but [Emacs] has an
+extension that add special support for F\*, including syntax highlighting and
+interactive development. More details on [editor support] on the [F\* wiki].
+You can find the code for the exercises and the solutions in the [F\* repository] 
+on Github. However, note that for some exercises, you have to include additional
+libraries. Details on how to do this for Emacs can be found in the wiki.
+
+~KK
+Both Atom and Vim support seems to be either broken or discontinued (or both),
+so it might be misleading to list them as viable options for the audience of this
+tutorial. Also, the code available on the tutorial page and on github differs
+quite a bit (as does the F\* version I guess). In my case, this lead to some
+unexpected errors when copying code from the online-editor to Emacs. It would be

```

Comment:
I guess that must have been it when I did the tutorial. I'll report it when I see it again (or hear from a student who does).